src/Core/Analytics/post-telemetry.ts

```diff
@@ -0,0 +1,58 @@
+import { NativeModules, Platform } from 'react-native';
+import { httpPost } from '../Services/https';
+import { AnalyticsProps } from './Analytics';
+const project = require('./../../../package.json');
+
+type Config = Pick<
+  AnalyticsProps,
+  'environment' | 'locale' | 'clientKey' | 'amount'
+>;
+
+const CHANNEL = Platform.select({
+  ios: () => 'iOS',
+  android: () => 'Android',
+  default: () => 'channel',
+})();
+
+const DEVICE_LOCALE = (
+  Platform.OS === 'ios'
+    ? NativeModules.SettingsManager.settings.AppleLocale ||
+      NativeModules.SettingsManager.settings.AppleLanguages[0] //iOS 13
+    : NativeModules.I18nManager.localeIdentifier
+).replace('_', '-');
+
+/**
+ * Log event to Adyen
+ * @param config -
+ */
+const logTelemetry = (config: Config) => (event: any) => {
+  if (!config.clientKey) return Promise.reject();
+
+  const options = {
+    errorLevel: 'silent' as const,
+    environment: config.environment,
+    path: `v2/analytics/log?clientKey=${config.clientKey}`,
+  };
+
+  const constants = Platform.constants as any;
+  const telemetryEvent = {
+    amountValue: config.amount?.value,
+    amountCurrency: config.amount?.currency,
+    version: project.version,
```

Comment:
Should be in another format.  Check [here](https://github.com/Adyen/adyen-web/pull/2219/files#diff-6594e49946aa8de03aa2a4f722d454f5c6ef7fcff93869f2e555d0c868ae166bR26)

Reply:
Fixed [41e150d](https://github.com/Adyen/adyen-react-native/pull/230/commits/41e150d4239834c6fb8046839fc49a7831066c09)